extensions/agroal/runtime-dev/src/main/java/io/quarkus/agroal/runtime/dev/ui/DatabaseInspector.java

```diff
@@ -223,7 +212,10 @@ public DataSet executeSQL(String datasource, String sql, Integer pageNumber, Int
                         }
                     }
                 } else {
-                    return new DataSet(null, null, "Only supported for Local Databases", null, -1);
+                    return new DataSet(null, null,
+                            "Datasource access not allowed. By default only local databases are enabled; you can use the"
+                                    + " 'quarkus.datasource.dev-ui.allowed-db-host' configuration property to configure allowed hosts ('*' to allow all).",
+                            null, -1);
```

Comment:
Great idea to improve the Agroal side as well; thanks for that!

Reply:
Of course, the setting is the same so we should keep consistent messaging.